src/components/Image/ImageLoader.jsx

```diff
@@ -12,20 +15,32 @@ const LOADED = 'LOADED'
 const FAILED = 'FAILED'
 
 class ImageLoader extends React.Component {
-  static contextTypes = {
-    client: PropTypes.object.isRequired
-  }
-
   state = {
     status: PENDING,
     src: null
   }
 
   _mounted = false
 
-  componentDidMount() {
+  async componentDidMount() {
+    const { client } = this.props
     this._mounted = true
     this.loadNextSrc()
+    this.realtime = new Realtime({ client })
+    this.type = 'io.cozy.files.thumbnails'
+    this.realtime.subscribe('created', this.type, this.handlerCreate)
+  }
+
+  /**
+   * Reload the link when realtime tell us that the
+   * thumbnail is created. By default size === small
+   */
+  handlerCreate = doc => {
```

Comment:
Nit: handleCreate or createHandler would be more correct english :)

Reply:
fixed (and also in our doc : https://github.com/cozy/cozy-libs/pull/855)